### PR TITLE
Fixes #574 Changes formatting of side card in search

### LINKED ISSUE #574: UI Enhancements
The indentation of the top right side content can be improved.
I will try to properly align it in a straight fashion.
I mean the content can be aligned  according to their type.

21755040_653125604893714_121576707_o 1

### PR DESCRIPTION
**Pull request by GingerDragon7**
I have done a minor UI enhancement 

Previous one:
screenshot from 2017-09-19 14-05-33

After Changes:
screenshot from 2017-09-19 15-29-53

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks! Please provide a link to a test page.

**Comment by hpdang** [comment]
Please read about our Best Practices. Add information about the issue you are working to the PR description. https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/

File: src/app/feed/info-box/info-box.component.html [hunk 1]
```diff
@@ -64,13 +64,22 @@
...
 </div>
 <div class="row link-sidebar-container">
-	<span class="link-sidebar"><a href="https://github.com/fossasia/loklak_search/issues/">Bug Report or Feature Request</a></span>
-	<span class="link-sidebar"><a href="http://fossasia.org">Developed by FOSSASIA</a></span>
-	<span class="link-sidebar"><a href="https://github.com/fossasia/loklak_search/graphs/contributors">Contributors</a></span>
-	<span class="link-sidebar"><a href="http://apps.loklak.org">Web Apps</a></span>
-	<span class="link-sidebar"><a href="https://play.google.com/store/apps/details?id=org.loklak.android.wok">loklak Android Wok</a></span>
-	<span class="link-sidebar"><a href="https://www.facebook.com/loklak.org/">Like on Facebook</a></span>
-	<span class="link-sidebar"><a href="https://twitter.com/loklak_">Follow on Twitter</a></span>
+	<ul>
```
Comment (djmgit) [inline src/app/feed/info-box/info-box.component.html]: @GingerDragon7 please indent all the <li> tags present after this.

**Comment by Achint08** [comment]
@GingerDragon7 Thanks for the PR. Please use a proper commit message describing what you have done in the commit, for eg, "Fixes `#{issue}- Describe here`. 
Use `npm run deploy` to deploy your own test application. 
Also, please solve the follow-up issues.

**Comment by mariobehling** [comment]
@GingerDragon7 Would be great if you would help and resolve this PR to get it merge ready. Keep it up!

**Comment by GingerDragon7** [comment]
hello everyone !!  
my test link -  https://gingerdragon7.github.io/loklak_search/

**Comment by Achint08** [comment]
Travis is failing. Please see.

**Comment by sudheesh001** [comment]
I just restarted the build, in case travis fails again, please fix the travis issue 😄 The rest of the code looks good for now. I am approving this patch but not the merge yet.

**Comment by mariobehling** [comment]
@GingerDragon7 Thanks. Please fix Travis fails. Cheers.

**Comment by mariobehling** [comment]
Merging this and we need travis fixes in following PRs. Cheers.

**Comment by mariobehling** [comment]
Sorry, I got confused. Actually this is not needed. Cheers.

**Pull request merged**